### PR TITLE
fixed: sequence dim for batch-first=False

### PR DESCRIPTION
**Pull request by pometa0507**
I modified it to follow [torch.nn.LSTM](https://pytorch.org/docs/stable/nn.html?highlight=lstm#torch.nn.LSTM).

### CODE REVIEW

**Comment by pometa0507** [comment]
A bug is  found.
